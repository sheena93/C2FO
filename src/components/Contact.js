/**
 * @component Contact
 * This component is a stateless component to show the about page
 * @return {element} returns Contact component
 */

import React from 'react';

const Contact = () => {
  return (
    <div className='container'>
      <h5>Contact Us</h5>
      <p>
      This Application show the book list with add and delete functionality.
      </p>
      <div className='divider' />
      <h6>Address:</h6>
      <ul>
        <li>2020 C2FO</li>
        <li>Noida</li>
      </ul>
    </div>
  );
};

export default Contact;