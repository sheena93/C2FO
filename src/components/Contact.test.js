import React from "react";
import {render,cleanup} from '@testing-library/react'
import Contact from './Contact';

afterEach(cleanup)

describe('Contact', () => {
    it('should take a snapshot', () => {
        const { asFragment } = render(<Contact />)
        expect(asFragment(<Contact />)).toMatchSnapshot()
       })
});



