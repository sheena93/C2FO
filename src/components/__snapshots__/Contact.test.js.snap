// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h5>
      Contact Us
    </h5>
    <p>
      This Application show the book list with add and delete functionality.
    </p>
    <div
      class="divider"
    />
    <h6>
      Address:
    </h6>
    <ul>
      <li>
        2020 C2FO
      </li>
      <li>
        Noida
      </li>
    </ul>
  </div>
</DocumentFragment>
`;

exports[`Contact should take a snapshot 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <h5>
      Contact Us
    </h5>
    <p>
      This Application show the book list with add and delete functionality.
    </p>
    <div
      class="divider"
    />
    <h6>
      Address:
    </h6>
    <ul>
      <li>
        2020 C2FO
      </li>
      <li>
        Noida
      </li>
    </ul>
  </div>
</DocumentFragment>
`;
